tp_timesheet/config.py

```diff
@@ -115,4 +119,13 @@ def _read_write_config(cls):
             logger.debug("Reading config file at: %s", cls.CONFIG_PATH)
         input_config = configparser.ConfigParser()
         input_config.read(cls.CONFIG_PATH)
+
+        # Version compatibility (#20)
+        for config_key in ["check_max_days", "max_submit_within_days"]:
```

Comment:
```suggestion
        for config_key in cls.DEFAULT:
```
Shouldn't have these values hard coded twice. Can iterate over the dictionary keys like this.